install/kubernetes/Makefile.digests

```diff
@@ -2,12 +2,12 @@
 # Copyright 2022 Authors of Cilium
 # SPDX-License-Identifier: Apache-2.0
 
-export CILIUM_DIGEST := "sha256:a732e57cb4881abe4783562bbba0045209ef85542372b44ce61584c887c49878"
```

Comment:
is this change intended ?

Reply:
I missed this comment, it's the same answer as below.